docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,563 @@
+      default: '20083232'
+      description: 'The per-structure seed for ancient cities.'
+    seed-buriedtreasure:
+      default: '10387320'
+      description: 'The per-structure seed for buried treasure.'
+    seed-desert:
+      default: '14357617'
+      description: 'The per-structure seed for desert structures.'
+    seed-endcity:
+      default: '10387313'
+      description: 'The per-structure seed for end cities.'
+    seed-fossil:
+      default: '14357921'
+      description: 'The per-structure seed for fossils.'
+    seed-igloo:
+      default: '14357618'
+      description: 'The per-structure seed for igloos.'
+    seed-jungle:
+      default: '14357619'
+      description: 'The per-structure seed for jungle structures.'
+    seed-mansion:
+      default: '10387319'
+      description: 'The per-structure seed for mansions.'
+    seed-mineshaft:
+      default: 'default'
+      description: 'The per-structure seed for mineshafts.'
+    seed-monument:
+      default: '10387313'
+      description: 'The per-structure seed for monuments.'
+    seed-nether:
+      default: '30084232'
+      description: 'The per-structure seed for the nether structures.'
+    seed-ocean:
+      default: '14357621'
+      description: 'The per-structure seed for the ocean structures.'
+    seed-outpost:
+      default: '165745296'
+      description: 'The per-structure seed for outposts.'
+    seed-portal:
+      default: '34222645'
+      description: 'The per-structure seed for portals.'
+    seed-shipwreck:
+      default: '165745295'
+      description: 'The per-structure seed for shipwrecks.'
+    seed-slime:
+      default: '987234911'
+      description: 'The per-structure seed for slime chunks.'
+    seed-stronghold:
+      default: 'default'
+      description: 'The per-structure seed for strongholds.'
+    seed-swamp:
+      default: '14357620'
+      description: 'The per-structure seed for swamp structures.'
+    seed-trailruins:
+      default: '83469867'
+      description: 'The per-structure seed for trail ruins.'
+    seed-village:
+      default: '10387312'
+      description: 'The per-structure seed for villages.'
+    simulation-distance:
+      default: 'default'
+      description: 'Overrides the simulation distance. Set to -1 or "default" to use the value in server.properties.'
+    thunder-chance:
+      default: '100000'
+      description: 'The chance of lightning occurring during a thunderstorm,
+        as a probability of 1/<thunder-chance> per chunk, every tick.'
+    ticks-per:
+      hopper-check:
+        default: '1'
+        description: 'The ticks between checks to pull items.'
+      hopper-transfer:
+        default: '8'
+        description: 'The ticks between hopper item movements.'
+    trident-despawn-rate:
+      default: '1200'
+      description: 'The number of ticks before a trident despawns.'
+    verbose:
+      default: 'false'
+      description: 'Whether to log world settings on startup.'
```

Comment:
This happens every time the config is loaded so not only on startup but also on /reload /spigot reload etc..